commands/run.go

```diff
@@ -39,26 +49,53 @@ var RunCmd = cli.Command{
 			return err
 		}
 
-		nodeRange, args := parseCommand(c.Args(), c.IsSet("terminator"))
-
-		if nodeRange == "" {
-			nodeRange = fmt.Sprintf("[0-%d]", len(nodes)-1)
+		var reader io.Reader
+		if c.IsSet("stdin") {
+			reader = bufio.NewReader(os.Stdin)
+		} else {
+			var builder strings.Builder
+			if c.IsSet("terminator") {
+				builder.WriteString("-- ")
+			}
+			for i, arg := range c.Args() {
+				builder.WriteString(arg)
+				if i != c.NArg()-1 {
+					builder.WriteString(" ")
+				}
+			}
+			reader = strings.NewReader(builder.String())
 		}
 
-		list, err := parseRange(nodeRange)
-		if err != nil {
-			return fmt.Errorf("could not parse node range %s", nodeRange)
+		var args [][]string
+		scanner := bufio.NewScanner(reader)
+		for scanner.Scan() {
+			tokens := strings.Fields(scanner.Text())
```

Comment:
This is actually bad idea -- for example, when you run `iptb run 0 -- sh -c 'echo hello >file'`, I think the `'echo hello >file'` portion needs to be passed as a single argument to the [this exec function](https://github.com/ipfs/iptb/blob/master/plugins/ipfs/docker/dockeripfs.go#L231). However, the `strings.Fields()` function splits on spaces, which means `'echo`, `hello`, and `>file'` get passed separately. I'm not quite sure how to handle this at the moment.

Reply:
cc'ing @travisperson for thoughts

Reply:
We should be able to use something like this: https://github.com/kballard/go-shellquote

Reply:
:+1: ended up going with https://github.com/mattn/go-shellwords instead (based on code/tests), same idea though.